src/Sdk.php

```diff
@@ -9,13 +9,23 @@
 /**
  * Creates a new Client and Hub which will be set as current.
  *
- * @param array $options The client options
+ * @param array $arrayOptions The client options
  */
-function init(array $options = []): void
+function init(array $arrayOptions = []): void
```

Comment:
Please revert this change and just override the variable later when wrapping it with the `Options` class